docs-content/index.html.md.erb

```diff
@@ -21,16 +21,16 @@ See which Azure services are supported below.
 
 Service modules with `Experimental` (Alpha) stability, which is intended for early feature review and evaluation:
 
-* [Azure CosmosDB](https://azure.microsoft.com/en-us/services/cosmos-db/)
 * [Azure Event Hubs](https://azure.microsoft.com/en-us/services/event-hubs/)
 * [Azure Key Vault](https://azure.microsoft.com/en-us/services/key-vault/)
 * [Azure Redis Cache](https://azure.microsoft.com/en-us/services/cache/)
 * [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/)
 * [Azure Storage](https://azure.microsoft.com/en-us/services/storage/)
+* [Azure Text Analytics (Cognitive Services)](https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/)
 
 Service modules with `Preview` (Beta) stability, which is intended for testing during proof of concept stage:
 
-* None
+* [Azure CosmosDB](https://azure.microsoft.com/en-us/services/cosmos-db/)
```

Comment:
Redis cache is also in preview now

Reply:
OK... seems in OSBA v1.2.0, Redis cache is not made preview,

Reply:
Yeah, Redis got preview after cutting v1.2.0.

Reply:
Hi @zhongyi-zhang 
Can you check the name of your product?
On the Azure site (https://azure.microsoft.com/en-us/services/cosmos-db/), I see it spelled "Azure Cosmos DB" but in these docs you've written "Azure CosmosDB". 
Probably, you want to check and correct this — gotta respect your product name.

Reply:
@jncd thanks for raising it. I'll create a PR to fix it soon.